addons/resource.language.en_gb/resources/strings.po

```diff
@@ -16541,3 +16541,13 @@ msgstr ""
 msgctxt "#38016"
 msgid "%d fps"
 msgstr ""
+
+#: system/settings/rbp.xml system/settings/imx6.xml
+msgctxt "#38120"
+msgid "Support 8 channel DTS HD audio decoding"
```

Comment:
Yeah - but you know what is best - I don't. I am always thankful for
comments, but sometimes - I am in a hurry and don't have the time. We
reworked some other language settings in the past together, which was quite
succesful, you remember?

If you give me two strings, I will PR them.

2015-05-10 14:31 GMT+02:00 da-anda <notifications@github.com>:

> In addons/resource.language.en_gb/resources/strings.po
> <https://github.com/xbmc/xbmc/pull/7102#discussion_r30001665>:
>
> > @@ -16541,3 +16541,13 @@ msgstr ""
> >  msgctxt "#38016"
> >  msgid "%d fps"
> >  msgstr ""
> > +
> > +#: system/settings/rbp.xml system/settings/imx6.xml
> > +msgctxt "#38120"
> > +msgid "Support 8 channel DTS HD audio decoding"
>
> I'm not THE entity for labels, so I only give suggestions. If you think
> they are fine I'll PR
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/xbmc/xbmc/pull/7102/files#r30001665>.
>



-- 
                   Key-ID:     0x1A995A9B
                   keyserver: pgp.mit.edu
==============================================================
Fingerprint: 4606 DA19 EC2E 9A0B 0157  C81B DA07 CF63 1A99 5A9B